tests/Fixtures/Symfony/Kernel.php

```diff
@@ -6,15 +6,15 @@
 use Symfony\Component\Config\Loader\LoaderInterface;
 use Symfony\Component\Config\Resource\FileResource;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\Routing\RouteCollectionBuilder;
+use Symfony\Component\Routing\Loader\Configurator\RoutingConfigurator;
 
 class Kernel extends \Symfony\Component\HttpKernel\Kernel
 {
     use MicroKernelTrait;
 
     const CONFIG_EXTS = '.{php,xml,yaml,yml}';
 
-    public function registerBundles()
```

Comment:
Does this change the php version?

Reply:
The neyt line‘s signature, sorry

Reply:
Hmm, I think these would need PHP 7.4? Again: as these were in the tests I worked on the basis that this should be OK. I was running against PHP 8 at the time. I can revert the changes but it's becoming increasingly difficult to have PHP <7.4 and PHP8 code in the same lib. SF6 adds return types to many classes which would then break these anyway so??? What would you prefer to do?

Reply:
I wouldn't know. But I agree- as these are only the tests it should be ok. Upgrading minimum php version would be a separate PR then.